api-tests/apply_route_test.go

```diff
@@ -332,6 +333,7 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 		{"latest", "1.7.0", nil, "4.4.3-5"},
 		{"latest", "1.6.0", nil, "4.4.2-4"},
 		{"latest", "1.5.0", nil, "4.2.8-8"},
```

Comment:
I look like we should have latest for 5.0 version too. For now we have only 4.2 4.4 6.0  latest.

Reply:
If it is just for some tests. This variant should be ok.